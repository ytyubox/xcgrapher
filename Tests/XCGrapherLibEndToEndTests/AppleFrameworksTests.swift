//
//  AppleFrameworksTests
//
//
//  Created by Yu Yu on 2023/4/10.
//

import ApprovalTests_Swift
import Foundation
@testable import XCGrapherLib
import XCTest

final class AppleFrameworksTests: XCTestCase {
  func test() throws {
    let all = try NativeDependencyManager().allNativeFrameworks.sorted { $0.lowercased() < $1.lowercased() }.joined(separator: "\n")
    try Approvals.verify(all)
  }
}
