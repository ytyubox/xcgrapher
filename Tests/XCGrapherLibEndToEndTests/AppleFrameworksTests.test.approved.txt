[000] = Accelerate
[001] = Accessibility
[002] = Accounts
[003] = ActivityKit
[004] = AddressBook
[005] = AddressBookUI
[006] = AdServices
[007] = AdSupport
[008] = AGL
[009] = AppClip
[010] = AppIntents
[011] = AppKit
[012] = AppleScriptKit
[013] = AppleScriptObjC
[014] = ApplicationServices
[015] = AppTrackingTransparency
[016] = ARKit
[017] = AssetsLibrary
[018] = AudioDriverKit
[019] = AudioToolbox
[020] = AudioUnit
[021] = AudioVideoBridging
[022] = AuthenticationServices
[023] = AutomatedDeviceEnrollment
[024] = AutomaticAssessmentConfiguration
[025] = Automator
[026] = AVFAudio
[027] = AVFoundation
[028] = AVKit
[029] = AVRouting
[030] = BackgroundAssets
[031] = BackgroundTasks
[032] = BlockStorageDeviceDriverKit
[033] = BusinessChat
[034] = CalendarStore
[035] = CallKit
[036] = Carbon
[037] = CarKey
[038] = CarPlay
[039] = CFNetwork
[040] = Charts
[041] = ClassKit
[042] = ClockKit
[043] = CloudKit
[044] = Cocoa
[045] = Collaboration
[046] = ColorSync
[047] = Combine
[048] = Contacts
[049] = ContactsUI
[050] = CoreAudio
[051] = CoreAudioKit
[052] = CoreAudioTypes
[053] = CoreBluetooth
[054] = CoreData
[055] = CoreDisplay
[056] = CoreFoundation
[057] = CoreGraphics
[058] = CoreHaptics
[059] = CoreImage
[060] = CoreLocation
[061] = CoreLocationUI
[062] = CoreMedia
[063] = CoreMediaIO
[064] = CoreMIDI
[065] = CoreMIDIServer
[066] = CoreML
[067] = CoreMotion
[068] = CoreNFC
[069] = CoreServices
[070] = CoreSpotlight
[071] = CoreTelephony
[072] = CoreText
[073] = CoreTransferable
[074] = CoreVideo
[075] = CoreWLAN
[076] = CreateML
[077] = CreateMLComponents
[078] = CryptoKit
[079] = CryptoTokenKit
[080] = DaemonUtils
[081] = DataDetection
[082] = DeveloperToolsSupport
[083] = DeviceActivity
[084] = DeviceCheck
[085] = DeviceDiscoveryExtension
[086] = DeviceDiscoveryUI
[087] = DirectoryService
[088] = DiscRecording
[089] = DiscRecordingUI
[090] = DiskArbitration
[091] = DriverKit
[092] = DVDPlayback
[093] = EventKit
[094] = EventKitUI
[095] = ExceptionHandling
[096] = ExecutionPolicy
[097] = ExposureNotification
[098] = ExtensionFoundation
[099] = ExtensionKit
[100] = ExternalAccessory
[101] = FamilyControls
[102] = FileProvider
[103] = FileProviderUI
[104] = FinderSync
[105] = ForceFeedback
[106] = Foundation
[107] = Frameworks
[108] = GameController
[109] = GameKit
[110] = GameplayKit
[111] = GLKit
[112] = GLUT
[113] = GroupActivities
[114] = GSS
[115] = HealthKit
[116] = HealthKitUI
[117] = HIDDriverKit
[118] = HomeKit
[119] = Hypervisor
[120] = iAd
[121] = ICADevices
[122] = IdentityLookup
[123] = IdentityLookupUI
[124] = ImageCaptureCore
[125] = ImageIO
[126] = IMServicePlugIn
[127] = InputMethodKit
[128] = InstallerPlugins
[129] = InstantMessage
[130] = Intents
[131] = IntentsUI
[132] = IOBluetooth
[133] = IOBluetoothUI
[134] = IOKit
[135] = IOSurface
[136] = IOUSBHost
[137] = iTunesLibrary
[138] = JavaNativeFoundation
[139] = JavaRuntimeSupport
[140] = JavaScriptCore
[141] = JavaVM
[142] = Kerberos
[143] = Kernel
[144] = KernelManagement
[145] = LatentSemanticMapping
[146] = LDAP
[147] = LinkPresentation
[148] = LocalAuthentication
[149] = LocalAuthenticationEmbeddedUI
[150] = MailKit
[151] = ManagedSettings
[152] = ManagedSettingsUI
[153] = MapKit
[154] = Matter
[155] = MatterSupport
[156] = MechanismBase
[157] = MediaAccessibility
[158] = MediaLibrary
[159] = MediaPlayer
[160] = MediaSetup
[161] = MediaToolbox
[162] = Message
[163] = Messages
[164] = MessageUI
[165] = Metal
[166] = MetalFX
[167] = MetalKit
[168] = MetalPerformanceShaders
[169] = MetalPerformanceShadersGraph
[170] = MetricKit
[171] = MLCompute
[172] = MobileCoreServices
[173] = ModelIO
[174] = ModuleBase
[175] = MultipeerConnectivity
[176] = MusicKit
[177] = NaturalLanguage
[178] = NearbyInteraction
[179] = NetFS
[180] = Network
[181] = NetworkExtension
[182] = NetworkExtensioniOSSupport
[183] = NetworkingDriverKit
[184] = NewsstandKit
[185] = NotificationCenter
[186] = OpenAL
[187] = OpenCL
[188] = OpenDirectory
[189] = OpenGL
[190] = OpenGLES
[191] = OSAKit
[192] = OSLog
[193] = ParavirtualizedGraphics
[194] = PassKit
[195] = PCIDriverKit
[196] = PCSC
[197] = PDFKit
[198] = PencilKit
[199] = PHASE
[200] = Photos
[201] = PhotosUI
[202] = PreferencePanes
[203] = ProximityReader
[204] = PushKit
[205] = PushToTalk
[206] = QTKit
[207] = Quartz
[208] = QuartzCore
[209] = QuickLook
[210] = QuickLookThumbnailing
[211] = QuickLookUI
[212] = RealityFoundation
[213] = RealityKit
[214] = ReplayKit
[215] = RoomPlan
[216] = Ruby
[217] = SafariServices
[218] = SafetyKit
[219] = SceneKit
[220] = ScreenCaptureKit
[221] = ScreenSaver
[222] = ScreenTime
[223] = ScriptingBridge
[224] = SCSIControllerDriverKit
[225] = SCSIPeripheralsDriverKit
[226] = SecureConfigDB
[227] = Security
[228] = SecurityFoundation
[229] = SecurityInterface
[230] = SensorKit
[231] = SerialDriverKit
[232] = ServiceManagement
[233] = SharedUtils
[234] = SharedWithYou
[235] = SharedWithYouCore
[236] = ShazamKit
[237] = Social
[238] = SoundAnalysis
[239] = Speech
[240] = SpriteKit
[241] = stageFramework.sh
[242] = StoreKit
[243] = StoreKitTest
[244] = SwiftUI
[245] = SyncServices
[246] = System
[247] = SystemConfiguration
[248] = SystemExtensions
[249] = TabularData
[250] = Tcl
[251] = ThreadNetwork
[252] = Tk
[253] = TVMLKit
[254] = TVServices
[255] = TVUIKit
[256] = TWAIN
[257] = Twitter
[258] = UIKit
[259] = UniformTypeIdentifiers
[260] = USBDriverKit
[261] = USBSerialDriverKit
[262] = UserNotifications
[263] = UserNotificationsUI
[264] = vecLib
[265] = VideoDecodeAcceleration
[266] = VideoSubscriberAccount
[267] = VideoToolbox
[268] = Virtualization
[269] = Vision
[270] = VisionKit
[271] = vmnet
[272] = WatchConnectivity
[273] = WatchKit
[274] = WeatherKit
[275] = WebKit
[276] = WidgetKit
[277] = XCTest
